src/jemalloc/src/jemalloc.c

```diff
@@ -1536,63 +1536,73 @@ je_pool_create(void *addr, size_t size, int zeroed)
 		return (NULL);
 	}
 
-	if (!zeroed)
-		memset(addr, 0, sizeof (pool_t));
+	if (empty) {
```

Comment:
Instead of indenting everything by one, I'd suggest moving these two operations to two distinct functions.